mediachain/transactor/client.py

```diff
@@ -38,23 +38,13 @@ def update_chain(self, cell, timeout=TIMEOUT_SECS):
         ref = self.client.UpdateChain(req, timeout)
         return MultihashReference.from_base58(ref.reference)
 
-    def journal_stream(self, last_block_ref=None, timeout=TIMEOUT_SECS):
-        if isinstance(last_block_ref, MultihashReference):
-            last_block_ref = last_block_ref.multihash_base58()
-
-        if last_block_ref is None:
-            req = Transactor_pb2.JournalStreamRequest()
-        else:
-            req = Transactor_pb2.JournalStreamRequest(
-                lastJournalBlock=Transactor_pb2.MultihashReference(
-                    reference=last_block_ref
-                ))
-            
+    def journal_stream(self, timeout=TIMEOUT_SECS):
+        req = Transactor_pb2.JournalStreamRequest()
         return self.client.JournalStream(req, timeout)
 
-    def canonical_stream(self, last_block_ref=None, timeout=TIMEOUT_SECS):
-        for event in self.journal_stream(last_block_ref, timeout):
-            if event.WhichOneof("event") == "chainUpdated":
+    def canonical_stream(self, timeout=TIMEOUT_SECS):
+        for event in self.journal_stream(timeout):
+            if event.WhichOneof("event") == "chainUpdateEvent":
```

Comment:
yeah, thanks for catching that.

This whole method should be re-written;  it's another quick-n-dirty demo hack.  Basically, the indexer wants to iterate over a stream of up-to-date canonicals, so we're only responding to the update event, and then it's making another trip to the transactor to get the chain head and fold over the chain.

So it's not close to a long-term solution 😄  We need to figure out the correct way to feed the indexer and have it respond to chain updates, which will probably involve removing this method.